azurerm/resource_arm_dns_ns_record_migration.go

```diff
@@ -0,0 +1,49 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func resourceArmDNSNsMigrateState(
```

Comment:
(as above) I think we can remove this, since this should be handled by a `refresh` which happens during a plan/apply?

Reply:
Done